packages/vector_graphics_compiler/test/test_svg_strings.dart

```diff
@@ -1010,6 +1015,85 @@ const String svgInlineImage = r'''
 </svg>
 ''';
 
+const String basicOverlap = '''
+<svg xmlns="http://www.w3.org/2000/svg" width="800" height="600">
+ <g id="Layer_1">
+  <rect id="svg_2" height="95" width="593" y="221.5" x="99" stroke-width="0" fill="red"/>
+  <rect  id="svg_4" height="485.99998" width="81" y="41.50001" x="367" fill="blue"/>
+ </g>
+</svg>
+''';
+
+const String opacityOverlap = '''
+<svg xmlns="http://www.w3.org/2000/svg" width="800" height="600">
+ <g id="Layer_1">
+  <rect opacity="0.5" id="svg_1" height="126" width="623" y="240.5" x="88" fill="#ff0000"/>
+  <rect opacity="0.3" id="svg_10" height="502.00001" width="141" y="63.5" x="343" fill="#0000ff"/>
+ </g>
+</svg>
+''';
+
+const String transparentOverSolid =
+    '''<svg xmlns="http://www.w3.org/2000/svg" width="800" height="600">
+ <g id="Layer_1">
+  <rect  id="svg_1" height="126" width="623" y="240.5" x="88" fill="#ff0000"/>
+  <rect  opacity="0.5" id="svg_10" height="502.00001" width="141" y="63.5" x="343" fill="#0000ff"/>
+ </g>
+
+</svg>''';
+
+const String solidOverTrasnparent =
+    ''' <svg xmlns="http://www.w3.org/2000/svg" width="800" height="600">
+ <g id="Layer_1">
+  <rect opacity="0.5" id="svg_1" height="126" width="623" y="240.5" x="88" fill="#ff0000"/>
+  <rect id="svg_10" height="502.00001" width="141" y="63.5" x="343" fill="#0000ff"/>
+ </g>
+
+</svg>''';
+
+const String complexOpacityTest =
+    '''<svg width="12cm" height="3.5cm" viewBox="0 0 1200 350"
+     xmlns="http://www.w3.org/2000/svg" version="1.1">
+
```

Comment:
Can you remove some of the extra whitespace here?

Reply:
fixed.